backoffice/src/main/java/com/wedriveu/backoffice/controller/RabbitmqInitialRequest.java

```diff
@@ -0,0 +1,38 @@
+package com.wedriveu.backoffice.controller;
+
+import com.wedriveu.services.shared.rabbitmq.VerticlePublisher;
+import com.wedriveu.shared.util.Constants;
+import com.wedriveu.shared.util.Log;
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+
+import static com.wedriveu.shared.util.Constants.ROUTING_KEY_ANALYTICS_REQUEST_VEHICLES;
+
+/**
+ * @author Stefano Bernagozzi
+ */
+public class RabbitmqInitialRequest extends VerticlePublisher{
```

Comment:
So, also here why this is a `controller`? Shouldn't be only the `boundary` to be able to communicate to the outside world through RabbitMQ?
Also, maybe `AnalyticsVehicleRequestPublisher` is better?